src/languages/es.js

```diff
@@ -304,6 +304,19 @@ export default {
         noPhoneNumber: 'Por favor escribe un número de teléfono que incluya el código de país e.g +447814266907',
         maxParticipantsReached: 'Has llegado al número máximo de participantes para un grupo.',
     },
+    beneficialOwnersStep: {
```

Comment:
It's occurred to me that I forgot to add the requestorStep values to this file 🙈